DESCRIPTION

```diff
@@ -21,7 +21,9 @@ Authors@R: c(
     person("Jojo", "Miller", role = "ctb"),
     person("Jayla", "Nakayama", role = "ctb"),
     person("Natalia", "Iannucci", email = "iannuccinatalia@gmail.com", role = "ctb"),
-    person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb"))
+    person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb"),
+    person("Anna", "Ballou", email = "aballou@smith.edu", role = "ctb")
```

Comment:
I recommend using an email you'll be keeping after you graduate. No worries if you'd rather not.

Reply:
Fixed!